samples/snippets/create_multiple_timeseries_forecasting_model.py

```diff
@@ -0,0 +1,106 @@
+def test_multiple_timeseries_forecasting_model(random_model_id):
+    # [START bigquery_dataframes_bqml_create_data__set]
+    your_model_id = random_model_id
+
+    from bigframes.ml import forecasting
+    import bigframes.pandas as bpd
+
+    # Start by selecting the data you'll use for training. `read_gbq_table` accepts
+    # either a SQL query or a table ID. Since this example selects from multiple
+    # tables via a wildcard, use SQL to define this data. Watch issue
+    # https://github.com/googleapis/python-bigquery-dataframes/issues/169
+    # for updates to `read_gbq_table` to support wildcard tables.
+
+    df = bpd.read_gbq_table("bigquery-public-data.new_york.citibike_trips", filters=[])
+
+    # [END bigquery_dataframes_bqml_create_data__set(1)]
+
+    # [START bigquery_dataframes_bqml_visualize_time_series_to_forecast]
```

Comment:
These two region tags can be combined since they are both for step 2.

Reply:
ok, makes sense! is "/ " or "&" the preference?